### PR TITLE
PMM-6834 : Change AWS staging Start job to use Amazon Linux 2 Spot Instance

### PR DESCRIPTION
**Pull request by AlyHKafoury**

### CODE REVIEW

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -477,9 +477,9 @@ pipeline {
                                     else
                                         pmm-agent setup --config-file=$PWD/pmm2-client/config/pmm-agent.yaml --server-address=\$IP:443 --server-insecure-tls --server-username=admin --server-password=admin \$IP
                                     fi
-                                    sleep 10
+                                    sudo sleep 10
```
Comment (BupycHuk) [inline pmm/aws-staging-start.groovy:480]: why just sleep doesn't work here?

Comment (EvgeniyPatlan) [reply pmm/aws-staging-start.groovy:480]: that is the main question. I assume PATH was missed somehow

File: pmm/aws-staging-start.groovy [hunk 2]
```diff
@@ -502,7 +502,7 @@ pipeline {
                             fi
                             [ -z "${CLIENTS}" ] && exit 0 || :
                             if [[ \$PMM_VERSION == pmm1 ]]; then
-                                bash /srv/pmm-qa/pmm-tests/pmm-framework.sh \
+                                sudo bash /srv/pmm-qa/pmm-tests/pmm-framework.sh \
```
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:505]: wouldn't it cause any further issues as we are going to run pmm-framework.sh under the root

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Comment by EvgeniyPatlan** [comment]
what ticket this review belongs to?

File: pmm/aws-staging-start.groovy (hunk 2 above)
Comment (AlyHKafoury) [reply pmm/aws-staging-start.groovy:505]: PMM-6834

Comment (puneet0191) [reply pmm/aws-staging-start.groovy:505]: I don't see the need to run pmm-framework.sh with sudo, why we need this?

**Pull request closed without merging**